### DISCUSSION TITLE
Feature Request: Compare by Filenames only

### DISCUSSION THREAD

**Discussion by richlux** [WinMerge/winmerge#1653] [Ideas]
I have a two huge directories of files that I want to compare by filename only.  The files were backed up at different times to different cloud services so the dates and times are different. In addition, for some reason, the file sizes in one backup are a few bytes smaller than the other backup, but the files open in their native apps and are identical.  Anyway, it would be nice if WinMerge could do a simple comparison by filename and/or extension only.  This would be an easy and fast way to see what files, by name, are missing between two folders.

Thanks,
Rich

**Comment by sdottaka** [comment]
If the two folders to be compared have the same structure, select a folder comparison method that does not compare file contents, such as "Modified Date and Size".
![image](https://user-images.githubusercontent.com/98126/211194996-db396745-d0ba-4f70-b5ae-79f3412ab9cf.png)

Unchecking the Show Identical Files and Show Different Files menu items as shown below will show only files that exist in only one folder.

![image](https://user-images.githubusercontent.com/98126/211195066-f877c515-9798-4698-bc94-57865f11376d.png)

**Reply by richlux** [reply]
As I mentioned, the files that I want to compare have different dates and times, so this won't work for me. The only type of comparison that will work in my situation would be a filename comparison.

Rich
